js/main.js

```diff
@@ -25,4 +25,81 @@ Pineapples can be consumed fresh, cooked, juiced, or preserved. They are found i
+	});
+
+	$('.orange').click(function() {
+		removeActive();
+		$(this).addClass('active');
+		$('#title').html('Orange');
+		$('#description').html('The orange is a hybrid between pomelo (Citrus maxima) and mandarin (Citrus reticulata). It has genes that are ~25% pomelo and ~75% mandarin; however, it is not a simple backcrossed BC1 hybrid, but hybridized over multiple generations. The chloroplast genes, and therefore the maternal line, seem to be pomelo. The sweet orange has had its full genome sequenced. Earlier estimates of the percentage of pomelo genes varying from ~50% to 6% have been reported.');
+	});
+
+	$('.peach').click(function() {
+		removeActive();
+		$(this).addClass('active');
+		$('#title').html('Peach');
+		$('#description').html('The peach (Prunus persica) is a deciduous tree native to the region of Northwest China between the Tarim Basin and the north slopes of the Kunlun Shan mountains, where it was first domesticated and cultivated. It bears an edible juicy fruit called a peach or a nectarine.');
+	});
+
+	$('.pear').click(function() {
+		removeActive();
+		$(this).addClass('active');
+		$('#title').html('Pear');
+		$('#description').html('The pear is any of several tree and shrub species of genus Pyrus, in the family Rosaceae. It is also the name of the pomaceous fruit of the trees. Several species of pear are valued for their edible fruit and juices, while others are cultivated as ornamental trees.');
+	});
+
+	$('.pineapple').click(function() {
+		removeActive();
+		$(this).addClass('active');
+		$('#title').html('Pineapple');
+		$('#description').html('Pineapples can be consumed fresh, cooked, juiced, or preserved. They are found in a wide array of cuisines. In addition to consumption, the pineapple leaves are used to produce the textile fiber piña in the Philippines, commonly used as the material for the men\'s barong Tagalog and women\'s baro\'t saya formal wear in the country. The fiber is also used as a component for wallpaper and other furnishings.');
+	});
+
+	$('.strawberry').click(function() {
+		removeActive();
+		$(this).addClass('active');
+		$('#title').html('Strawberry');
+		$('#description').html('The garden strawberry (or simply strawberry; Fragaria × ananassa) is a widely grown hybrid species of the genus Fragaria (collectively known as the strawberries). It is cultivated worldwide for its fruit. The fruit (which is not a botanical berry, but an aggregate accessory fruit) is widely appreciated for its characteristic aroma, bright red color, juicy texture, and sweetness. It is consumed in large quantities, either fresh or in such prepared foods as preserves, fruit juice, pies, ice creams, milkshakes, and chocolates. Artificial strawberry flavorings and aromas are also widely used in many products like lip gloss, candy, hand sanitizers, perfume, and many others.');
+	});
+})
```

Comment:
While its not a _hard_ requirement, Its best to put a `;` at the end here (e.g. `});`